components/Pledge/LegalOverlay.js

```diff
@@ -13,6 +13,12 @@ import {
 
 import { MdClose } from 'react-icons/md'
 import Loader from '../Loader'
+import { compose, graphql } from 'react-apollo'
+import { getDocument } from '../Article/graphql/getDocument'
+import { splitByTitle } from '../../lib/utils/mdast'
+import { renderMdast } from 'mdast-react-render'
+import createPageSchema from '@project-r/styleguide/lib/templates/Page'
+import Campaign from '../Access/Campaigns/Campaign'
```

Comment:
your editor went rough

Reply:
:D I copy pasted the loader from somewhere